Firestore/core/src/firebase/firestore/objc/objc_class.h

```diff
@@ -159,6 +164,7 @@ bool Equals(const Handle<T>& lhs, const Handle<T>& rhs) {
 #else  // !__clang__
 #define NS_ASSUME_NONNULL_BEGIN
 #define NS_ASSUME_NONNULL_END
+#define _Nullable
```

Comment:
Question: I remember we discussed putting the definition of `_Nullable` in a header or something like that, can you remind me what was the outcome of that?

Reply:
I'm fine with the current state, just checking.